lib/exporter/EnvFileExporter_1_0_0.js

```diff
@@ -15,15 +15,16 @@
 
 const async = require('async');
 const fs = require('fs');
-const _ = require('lodash');
+const isString = require('lodash.isstring');
+const findKey = require('lodash.findkey');
 
 
 const Utils = require('../Utils');
 
 const { ActiveItemType, AppOptionsName, CommonOptionsNames, ConfigType, EntityType, EnvOptionsName, Errors, Mapping, OperationType, BackendCollectionPermission, DomainTypes } = require('../Constants');
 const KinveyError = require('../KinveyError');
 
-class EnvFileExporter_1_0_0 {
+class EnvFileExporterV1 {
```

Comment:
File should be renamed too -> https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#naming--filename-matches-export

Reply:
Well, I hoped to leave the file like that, but since it is official, I will rename it as well.

Initially I thought of the schema version as a "1.0.0" type of versioning. While renaming the classes, I thought that we might want to just have a single number version for schema version.